gulpfile.js

```diff
@@ -143,9 +143,7 @@ var jsTasks = function(filename) {
 var writeToManifest = function(directory) {
   return lazypipe()
     .pipe(gulp.dest, path.dist + directory)
-    .pipe(function() {
-      return gulpif('**/*.{js,css}', browserSync.reload({stream:true}));
-    })
+    .pipe(browserSync.stream, {match: '**/*.{js,css}'})
```

Comment:
I don't think that really works. I'm seeing it picking up every changed file in `dist`, including source maps. The additional files mess up BrowserSync's asset-injection and force a full page reload. With a lot of files, they can also send multiple reload triggers. 

![pastedgraphic-2](https://cloud.githubusercontent.com/assets/8320/7506814/38e001f0-f433-11e4-98fc-e7b4e908640d.png)